Move EMAN related terms from Framework to Terminololgy draft (#56)
billwuqin/draft-bclp-green-terminology: draft-ietf-green-terminology.md

: Refers to the time rate at which energy is emitted, transferred, or
  received; power is usually expressed in watts (joules per second) ({{?RFC7326}}).

Device:

> : A device is a piece of electrical or non-electrical equipment.

Reviewer: ```suggestion
: A piece of electrical or non-electrical equipment.
```

What is meant by "piece" here?

Author: it means one single or individual item. It turns an uncountable noun (equipment) into a single, countable unit.
The original defintion comes from GREEN Framework.

> Reference: Adapted from {{IEEE100}}.

Reviewer: ```suggestion
Adapted from {{IEEE100}}.
```

Component:

> : A component is a part of electrical or non-electrical equipment

Reviewer: ```suggestion
: Is a part of electrical or non-electrical equipment
```

(device).

> Reference: Adapted from {{TMN}}.

Reviewer: ```suggestion
Adapted from {{TMN}}.
```

Meter (Energy Meter):

> : A meter is a device intended to measure electrical energy by

Reviewer: ```suggestion
: Is a device/component that is intended to measure electrical energy by
```

integrating power with respect to time.

> Reference: Adapted from {{IEC60050}}.

Reviewer: ```suggestion
Adapted from {{IEC60050}}.
```

> Power Inlet:

Reviewer: ```suggestion
Power Inlet (inlet, for short):
```

> : A power inlet (or simply "inlet") is an interface at which a

Reviewer: ```suggestion
: Is an interface at which a
```

> device or component receives energy from another device or

Reviewer: ```suggestion
device  (or component) receives energy from another device  (or component)
```

> component ({{?RFC7326}}).

Reviewer: ```suggestion
{{?RFC7326}}.
```

> Power Outlet:

Reviewer: ```suggestion
Power Outlet (outlet, for short):
```

> : A power outlet (or simply "outlet") is an interface at which a

Reviewer: ```suggestion
: Is an interface at which a
```

> device or component provides energy to another device or

Reviewer: ```suggestion
device  (or component)  provides energy to another device  (or component)
```

> component ({{?RFC7326}}).

Reviewer: ```suggestion
{{?RFC7326}}.
```

Power Interface:

> : A Power Interface is a power inlet, outlet, or both ({{?RFC7326}}).

Reviewer: ```suggestion
: Is a power inlet, outlet, or both {{?RFC7326}}.
```

Power State:

> : A Power State is a condition or mode of a device (or component)

Reviewer: ```suggestion
: Is a condition or mode of a device (or component)
```

that broadly characterizes its capabilities, power, and
responsiveness to input.

> Reference: Adapted from [IEEE1621].

Reviewer: ```suggestion
Adapted from [IEEE1621].
```

Power State Set:

> : A Power State Set is a collection of Power States that comprises a

Reviewer: ```suggestion
: Is a collection of Power States that comprises a
```
